recipe/meta.yaml

```diff
@@ -28,9 +28,6 @@ requirements:
     - tbb
     - tbb-devel
 
-  run:
-    - tbb
```

Comment:
Have you tested builds without tbb installed?

I include it as a runtime dependency because `libembree` is linked to `libtbb`
```
$ ldd ~/miniforge3/lib/libembree4.so
	linux-vdso.so.1 (0x00007ffed0b01000)
	libtbb.so.12 => /home/joaander/miniforge3/lib/./libtbb.so.12 (0x00007fae176e3000)
	libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007fae176c8000)
	libstdc++.so.6 => /home/joaander/miniforge3/lib/./libstdc++.so.6 (0x00007fae1581d000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fae15736000)
	libgcc_s.so.1 => /home/joaander/miniforge3/lib/./libgcc_s.so.1 (0x00007fae176ad000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fae15400000)
	/lib64/ld-linux-x86-64.so.2 (0x00007fae17740000)
	libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007fae176a6000)
	librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007fae176a1000)
```
The documentation states that methods including `rtcCommitScene` operate in parallel using TBB.

Reply:
Yes that’s all good as tbb-devel has a run_export on tbb, so tbb will still be installed at run time. At the moment we’re doubling up. Furthermore, thankfully the conda linker check would catch the case of a missing run dependency. So we’re all good.

Reply:
Understood.